front/app/services/events.ts

```diff
@@ -47,8 +47,9 @@ export interface IUpdatedEventProperties {
 export type IEventsStreamParams = IStreamParams & {
   queryParameters: {
     project_ids?: string[];
-    start_at_lt?: string;
-    start_at_gteq?: string;
+    end_at_lt?: string;
+    end_at_gteq?: string;
+    sort?: string;
```

Comment:
Can we add the different strings we can expect for `sort`?

Reply:
Could be even more readable.

ends_before_date
ends_on_or_after_date

Reply:
I would say it doesn't matter, but for me `lt` and `qteq` are very readable and look similar to `end_at >= ...`.

Reply:
Already changed.